samples/sampler/stories/default/ProgressBar.js

```diff
@@ -1,46 +1,49 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {boolean, select, number} from '@enact/storybook-utils/addons/knobs';
 import UiProgressBar from '@enact/ui/ProgressBar';
-import {storiesOf} from '@storybook/react';
 
 import ProgressBar, {ProgressBarBase, ProgressBarTooltip} from '@enact/agate/ProgressBar';
 
 const ProgressBarConfig = mergeComponentMetadata('ProgressBar', ProgressBarBase, ProgressBar, UiProgressBar);
 const ProgressBarTooltipConfig = mergeComponentMetadata('ProgressBarTooltip', ProgressBarTooltip);
 
-ProgressBar.displayName = 'ProgressBar';
 ProgressBarTooltip.displayName = 'ProgressBarTooltip';
```

Comment:
```suggestion
```

Reply:
Maybe this should not be deleted, this line is now present on develop branch